native/ffi.cc

```diff
@@ -49,6 +49,64 @@ Napi::Value PactffiInitWithLogLevel(const Napi::CallbackInfo& info) {
   return env.Undefined();
 }
 
+LevelFilter integerToLevelFilter(Napi::Env &env, uint32_t number) {
+  LevelFilter level = LevelFilter::LevelFilter_Off;
+
+  switch(number) {
+    case 0:
+      level = LevelFilter::LevelFilter_Off;
+      break;
+    case 1:
+      level = LevelFilter::LevelFilter_Error;
+      break;
+    case 2:
+      level = LevelFilter::LevelFilter_Warn;
+      break;
+    case 3:
+      level = LevelFilter::LevelFilter_Info;
+      break;
+    case 4:
+      level = LevelFilter::LevelFilter_Debug;
+      break;
+    case 5:
+      level = LevelFilter::LevelFilter_Trace;
+      break;
+    default:
+      std::string err =  "Unable to parse Level Filter: ";
```

Comment:
This would be better if it included more about the source (eg, pact-js-core C integration, and "log level" instead of "Level")

Reply:
agreed 👍🏾

Reply:
Updated to

```
    case 5:
      logLevel = LevelFilter::LevelFilter_Trace;
      break;
    default:
      std::string err =  "pact-js-core C integration: Unable to parse log level number: ";
      err += number;
      throw Napi::Error::New(env, err);
```